__tests__/index.test.js

```diff
@@ -20,4 +20,57 @@ describe('/api/topics', () => {
           });
         });
     });
-})
\ No newline at end of file
+})
+
+describe('/api/invalid-endpoint', () => {
+  test('GET:404 for invalid endpoint', () => {
+    return request(app)
+      .get('/api/invalid-endpoint')
+      .expect(404)
+      .then((response) => {
+          expect(response.body.msg).toBe('Invalid endpoint');
+      });
+  });
+})
+
+describe('/api', () => {
+  test('GET:200 Responds with an object describing all the available endpoints', () => {
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then((response) => {
+          const data = response.body.endpointData
+          const expectedOutput = {
```

Comment:
This test works now, but when you add more endpoints in the future you'd need to come back and edit this.

Instead of comparing the response to this copy output, just require the endpoints object into the test file, and assert that the response looks like what you've required in :)

Reply:
Ok, sure,  will do.